I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 5, in script
    define stage = start
  File "game/script.rpy", line 5, in <module>
    define stage = start
NameError: name 'start' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 5, in script
    define stage = start
  File "C:\Users\User\Desktop\renpy-8.2.1-sdk\renpy\ast.py", line 2219, in execute
    self.set()
  File "C:\Users\User\Desktop\renpy-8.2.1-sdk\renpy\ast.py", line 2233, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\User\Desktop\renpy-8.2.1-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 5, in <module>
    define stage = start
NameError: name 'start' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.1.24030407
MSv1 1.0
Sun Oct 20 09:41:08 2024
